core/certdb.go

```diff
@@ -277,13 +277,46 @@ func (d *CertDb) addPhishletCertificate(site_name string, base_domain string, ce
 	d.phishletCache[base_domain][site_name] = cert
 }
 
-func (d *CertDb) loadPhishletCertificate(site_name string, base_domain string) error {
+func (d *CertDb) loadPhishletCertificate(site_name string, base_domain string, domains []string) error {
 	crt_dir := filepath.Join(d.dataDir, base_domain)
 
 	cert, err := tls.LoadX509KeyPair(filepath.Join(crt_dir, site_name+".crt"), filepath.Join(crt_dir, site_name+".key"))
 	if err != nil {
 		return err
 	}
+
+	cert_x509, err := x509.ParseCertificate(cert.Certificate[0])
```

Comment:
Could it be useful to loop through cert.Certificate instead of hardcoding the 1st one? I'm not sure, but I looped on all available certificate on my local fix, but don't know if it's necessary.

Reply:
Hi! is not needed because LetsEncrypt will generate just one cert, not the full chain.
is not wrong to loop though